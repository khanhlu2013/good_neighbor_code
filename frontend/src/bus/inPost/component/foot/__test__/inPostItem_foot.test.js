import React from "react";
import { shallow } from "enzyme";
import InPostItemFoot from "../inPostItem_foot";

describe("inPostItem_foot", () => {
  it("render request post foot correctly", () => {
    const wrap = shallow(
      <InPostItemFoot
        postId="1"
        isActive={true}
        myBorrowShareId={null}
        myRequestShareId={"2"}
      />
    );
    expect(wrap).toMatchSnapshot();
  });

  it("render approve post foot correctly", () => {
    const wrap = shallow(
      <InPostItemFoot
        postId="1"
        isActive={true}
        myBorrowShareId={"3"}
        myRequestShareId={null}
      />
    );
    expect(wrap).toMatchSnapshot();
  });

  it("render non-request and non-approve and active post foot correctly", () => {});

  it("render non-request and non-approve and non-active post foot correctly", () => {});
});
