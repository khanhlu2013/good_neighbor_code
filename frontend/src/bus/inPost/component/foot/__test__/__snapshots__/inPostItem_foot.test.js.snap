// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`inPostItem_foot render approve post foot correctly 1`] = `
<styled.div>
  <Connect(InPostItemFootBorrow)
    myBorrowShareId="3"
  />
</styled.div>
`;

exports[`inPostItem_foot render request post foot correctly 1`] = `
<styled.div>
  <Connect(InPostItemFootRequest)
    myRequestShareId="2"
  />
</styled.div>
`;
